.github/workflows/test.yml

```
@@ -0,0 +1,29 @@
+name: Ruby CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: [2.4, 2.5, 2.6, 2.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install OS dependencies
+      run: |
+        sudo apt-get install libcurl4-openssl-dev
```

```suggestion
        sudo apt-get install --assume-yes libcurl4-openssl-dev
```

Perhaps we want something like that for it be non-interactive?

> One side effect of using `-y` is that it will skip downgrades, whereas if the user entered y, it would perform the downgrades. So if you want `-y` to execute the same thing as a user entering y, you also need to add `--allow-downgrades`.

But we don't need downgrades.